rosetta-runtime/src/main/java/com/rosetta/model/lib/expression/ExpressionOperators.java

```diff
@@ -47,7 +47,7 @@ public static <T> ComparisonResult exists(Mapper<T> o) {
 			return ComparisonResult.success();
 		}
 		String failureMessage = null;
-		if (o.getErrorPaths()!= null && !o.getErrorPaths().isEmpty()){
+		if (o.getErrorPaths()!= null && !o.getErrorPaths().isEmpty() && !o.getErrorPaths().toString().contains("Null")){
```

Comment:
Q: why are some error paths null? This seems a bit of a hack to workaround a bug. It is probably better to find out why we are generating error paths containing a string "Null".

Reply:
these are coming from when condition is within a function which is passed a  value like x-> y-> z-> and it return the last object as "null" as it doesn't know which object was actually passed.

Reply:
o.getErrorPaths()!= null && !o.getErrorPaths().isEmpty() this did not return true and its was not empty or null but rather it had value "Null"